presto-main/src/main/java/com/facebook/presto/server/GracefulShutdownHandler.java

```diff
@@ -174,6 +211,11 @@ private void waitForTasksToComplete()
         }
     }
 
+    public AtomicBoolean getNoTaskAtGracefulShutdown()
```

Comment:
nit: Should we avoid "negations" in return values -- isAnyTaskAtGracefulShutdown ?